static/templates/compose_all_everyone.handlebars

```diff
@@ -1,8 +1,8 @@
 <div class="compose-all-everyone">
     <span class="compose-all-everyone-msg">
-        {{#tr this}}Are you sure you want to message all <strong>__count__</strong> people in this stream?{{/tr}}
+        {{#tr this}}Are you sure you want to mention/message all <strong>__count__</strong> people in this stream?{{/tr}}
```

Comment:
We should just replace "message" with "mention".  Also, I think it deserves its own commit, so I went ahead and just merged that change to master.